### PR TITLE
[IOTDB-5616] Fix some code smells

### PR DESCRIPTION
**Pull request by l2280212**
fix some code smells，please see in [JIRA5616](https://issues.apache.org/jira/browse/IOTDB-5616).

### CODE REVIEW

File: confignode/src/main/java/org/apache/iotdb/confignode/consensus/statemachine/ConfigRegionStateMachine.java [hunk 1]
```diff
@@ -364,7 +364,10 @@ private void flushWALForSimpleConsensus() {
   private void createLogFile(int endIndex) {
     simpleLogFile = SystemFileFactory.INSTANCE.getFile(PROGRESS_FILE_PATH + endIndex);
     try {
-      simpleLogFile.createNewFile();
+      if (!simpleLogFile.createNewFile())
```
Comment (OneSizeFitsQuorum) [inline confignode/src/main/java/org/apache/iotdb/confignode/consensus/statemachine/ConfigRegionStateMachine.java:367]: add `{` and `}`

Comment (l2280212) [reply confignode/src/main/java/org/apache/iotdb/confignode/consensus/statemachine/ConfigRegionStateMachine.java]: fixed

**Review by OneSizeFitsQuorum — APPROVED** [review]
LGTM

**Review by SpriCoder — APPROVED** [review]
LGTM~

**Pull request merged**